test/test_modules_type_parser.rb

```diff
@@ -16,8 +17,6 @@ def examples
         "n"=>"n",
         "12 bomen"=>"12 bomen",
         "12 bomenasdfasdfsadf"=>"12 bomenasdfasdfsadf",
-        ""=>nil,
-        " "=>nil,
```

Comment:
Ah, now I remember: sometimes excel files are full of crap ('empty'-cells) until the end of the file. I trim the files to make them convenient to handle.